migrate/v0_15/migrate.go

```diff
@@ -174,11 +175,38 @@ func Committee(genesisState v0_14committee.GenesisState) v0_15committee.GenesisS
 			baseStabilityCom := stabilityCom.SetPermissions(newStabilityCommitteePermissions)
 			newStabilityCom := v0_15committee.MemberCommittee{BaseCommittee: baseStabilityCom}
 			committees = append(committees, newStabilityCom)
-		} else {
+		case 2:
 			safetyCom := comtypes.NewMemberCommittee(com.ID, com.Description, com.Members,
 				[]v0_15committee.Permission{v0_15committee.SoftwareUpgradePermission{}},
 				com.VoteThreshold, com.ProposalDuration, v0_15committee.FirstPastThePost)
 			committees = append(committees, safetyCom)
+		case 3:
+			// Initialize swap governance committee without permissions
+			quorum := sdk.MustNewDecFromStr("0.33")
+			tallyDenom := "swp"
+			swapGovCom := comtypes.NewTokenCommittee(com.ID, com.Description, com.Members,
+				[]v0_15committee.Permission{}, com.VoteThreshold, com.ProposalDuration,
+				v0_15committee.FirstPastThePost, quorum, tallyDenom)
+
+			// Build swap governance committee permissions
+			var newSwapCommitteePermissions []v0_15committee.Permission
+			var newSwapSubParamPermissions v0_15committee.SubParamChangePermission
+			for _, perm := range com.Permissions {
```

Comment:
Same for this - will have to build permissions manually outside of switch and set them on the newly created committee.

Reply:
https://github.com/Kava-Labs/kava/pull/940/commits/084a00790d38befdadf211fea012df88c0d6f6df